Documentation/parameters.par

```diff
@@ -77,6 +78,10 @@ PPFileName = 'tpv33_faultreceivers.dat'
 &MeshNml
 MeshFile = 'tpv33_gmsh'         ! Name of mesh file
 meshgenerator = 'PUML'          ! Name of meshgenerator (Gambit3D-fast, Netcdf or PUML)
+ertexWeightElement = 100 ! Base vertex weight for each element used as input to ParMETIS
+vertexWeightDynamicRupture = 200 ! Weight that's added for each DR face to element vertex weight
+vertexWeightDisplacement = 300 ! Weight that's added for each free surface with gravity face to element vertex weight
```

Comment:
Maybe displacement is the wrong term here. At an elastic acoustic interface or at a normal free surface, we also compute displacements. Might confuse users with this naming

Reply:
True. It actually confused me. The original intention was that we weight each face which has an displacement. (They were originally updated with the same solver). I actually implemented it like you suggest. So slightly underreporting the FLOPS.